Backend/ServiceLayer/Services/ResetService.cs

```diff
@@ -0,0 +1,229 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+using DataAccessLayer.Database;
+using DataAccessLayer.Models;
+using DataAccessLayer.Repositories;
+using MimeKit;
+using MailKit;
+using MailKit.Net.Smtp;
+
+
+namespace ServiceLayer.Services
+{
+    public class ResetService
+    {
+
+        #region Constants for the reset service
+        //Variable for how long the token is supposed to be live
+        private const double TimeToExpire = 5;
+        //Constant for the password reset email subject line
+        private const string resetPasswordSubjectString = "KFC SSO Password Reset";
+        //URL to append the reset ID to. Sent to the user in the email body.
+        private const string resetControllerURL = "kfcsso.com/api/reset/resetpassword/?id=";
+
+        #endregion
+
+        private ResetRepository _resetRepo;
+        public ResetService()
+        {
+            _resetRepo = new ResetRepository(new DatabaseContext());
+        }
```

Comment:
DatabaseContext is created in manager layer so that changes may be rolled back after several services are called

Reply:
This applies to a comment I made above